spec/p2p/messages/mempool.md

```diff
@@ -14,7 +14,7 @@ Mempool has one channel. The channel identifier is listed below.
 ## Message Types
 
 There is currently only one message that Mempool broadcasts and receives over
-the p2p gossip network (via the reactor): `TxsMessage`
+the p2p gossip network (via the reactor): `Message`
```

Comment:
Thanks! I prefer to leave the clarification of the wrap/unwrap methods for another PR as it's not specific to the mempool and it would require better understanding of how it works.